namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java

```diff
@@ -455,6 +470,10 @@ private void createAndUpdateQueueData(final String brokerName, final TopicConfig
                 log.info("topic changed, {} OLD: {} NEW: {}", topicConfig.getTopicName(), existedQD,
                     queueData);
                 queueDataMap.put(brokerName, queueData);
+                // if broker restart, many topic will register, for avoid hot, then ignore
+                if (singleTopicRouteChanged) {
```

Comment:
In case of "broker restart/failure", it also needs to be notified.

How about using the multi-node to improve performance and reduce CPU cost?